Sources/TSCBasic/Path.swift

```diff
@@ -534,36 +513,17 @@ private struct WindowsPath: Path, Sendable {
         return String(cString: representation)
     }
 
-    init(normalizingAbsolutePath path: String) {
-        self.init(string: Self.repr(path).withCString(encodedAs: UTF16.self) { pwszPath in
-          var canonical: PWSTR!
-          _ = PathAllocCanonicalize(pwszPath,
-                                    ULONG(PATHCCH_ALLOW_LONG_PATHS.rawValue),
-                                    &canonical)
-          return String(decodingCString: canonical, as: UTF16.self)
-        })
-    }
-
```

Comment:
Restoring this enables building the library, was this intentionally removed?

Reply:
sure, is this windows only?

Reply:
for reference, it was removed because hit was removed from the Path API - the whole idea you pushed for was  that we dont normalize / canonicalize  on construction because that cant work correctly on windows

Reply:
in any case, I am happy to restore if that is what you want for windows, it just seems like it would be defeating the purpose of these changes

Reply:
This isn't just Windows specific - it seems that there are references to the APIs on Windows and Unix.  I agree, it does effectively defeat the purpose of the change.  Without these two functions though, we are unable to build t-s-c with this change.  It may be that you had some other changes that got left out?